docs/reference/modules.md

```
@@ -81,37 +81,45 @@ For each search path, the system checks:
-fn _helper() -> int:     # private — not importable
-    return 42
-fn publicApi() -> int:   # public — importable
-    return _helper()
+fn helper() -> int:           # package-internal — visible only from the
+    return 42                 # same package as mylib/internal.ry
+@public
+fn publicApi() -> int:        # public — visible from any package
+    return helper()
 ```
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)